pandas/core/arrays/sparse/array.py

```diff
@@ -1342,7 +1353,8 @@ def __setstate__(self, state):
         if isinstance(state, tuple):
             # Compat for pandas < 0.24.0
             nd_state, (fill_value, sp_index) = state
-            sparse_values = np.array([])
+            # Need type annotation for "sparse_values"  [var-annotated]
+            sparse_values = np.array([])  # type: ignore[var-annotated]
```

Comment:
I think it might be better to write explicit type annotations instead of ignoring `var-annotated` otherwise mypy will treat it as `Any`(?).

Reply:
See above